CONTRIBUTING.md

```diff
@@ -139,12 +136,13 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
 
 You can add the sign off when creating the git commit via `git commit -s`.
 
-[code-of-conduct]: https://github.com/opencontainers/tob/blob/d2f9d68c1332870e40693fe077d311e0742bc73d/code-of-conduct.md
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/.github/CODE_OF_CONDUCT.md
```

Comment:
This is now at the root `https://github.com/opencontainers/org/blob/master/CODE_OF_CONDUCT.md`

Reply:
Just fixed, per our discussion in the OCI call!